posthog-react-native/src/PosthogContext.ts

```diff
@@ -1,4 +1,4 @@
 import React from 'react'
 import { PostHog } from './posthog-rn'
 
-export const PostHogContext = React.createContext<{ client?: PostHog }>({ client: undefined })
+export const PostHogContext = React.createContext<{ client: PostHog }>({ client: undefined as unknown as PostHog })
```

Comment:
Can we just remove `{ client: undefined as unknown as PostHog }`? since its declared as non nullable/undefined?

Reply:
unfortunately not. The argument requires the type to match (which undefined doesn't). It's one of the quirks of context hooks sadly.